sjb/config/test_cases/push_origin_release.yml

```diff
@@ -46,4 +47,10 @@ extensions:
       repository: "origin-web-console-server"
       script: |-
         DOCKER_CONFIG=/tmp/.docker OS_PUSH_LOCAL=1 OS_PUSH_ALWAYS=1 OS_PUSH_BASE_REGISTRY="docker.io/" hack/push-release.sh
-        sudo rm -rf /tmp/.docker
\ No newline at end of file
+        sudo rm -rf /tmp/.docker
```

Comment:
move this line into its own step at the bottom, or you're going to nuke your docker credentials before you try to push